rpc-client-interface.go

```diff
@@ -63,4 +63,9 @@ type NetLocker interface {
 
 	// Return this lock server service endpoint on which the server runs.
 	ServiceEndpoint() string
+
+	// Init attempts to initialize the underlying remote connection.
+	// if already initialized closes the connection and re-connects,
+	// if not initializes a new rpc connection.
+	Init() error
```

Comment:
This `Init()` is nowhere used in dsync.  I am unable to understand the need WRT dsync.

As `NetLocker` is an interface, its up to implementor to do any other additional logic for any such method.